components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.publisher/src/main/java/org/wso2/micro/integrator/prometheus/publisher/util/MetricFormatter.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.micro.integrator.prometheus.publisher.util;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.Collector.MetricFamilySamples;
+import io.prometheus.client.Collector.Type;
+
+import java.util.Enumeration;
+
+/**
+ * Metric Formatter class to format Prometheus Metrics
+ */
+public class MetricFormatter {
+
+  /**
+     * Return the type of metric
+     *
+     * @param type Metric Prometheus type
+   */
+ private String typeString(Type type) {
+
+        switch (type) {
+            case GAUGE:
+                return "gauge";
+            case COUNTER:
+                return "counter";
+            case SUMMARY:
+                return "summary";
+            case HISTOGRAM:
+                return "histogram";
+            default:
+                return "untyped";
+        }
+    }
+
+    /**
+     * Return a formatted metric
+     *
+     * @param mfs the Prometheus Metric from Prometheus Default Registry
+     */
+    public StringBuilder formatMetrics(Enumeration<MetricFamilySamples> mfs) {
```

Comment:
lest's have a self explainatory name instead of 'mfs'

Reply:
Fixed